src/plots/smith/helpers.js

```diff
@@ -0,0 +1,253 @@
+'use strict';
```

Comment:
It looks there is no need for this file. If so you could reuse `src/plots/polar/helpers.js` where needed.

Reply:
5c83a18b0743b421fcacb56c5d660aff88f38353